src/states/node.rs

```diff
@@ -871,8 +871,8 @@ impl Node {
                                                      src_name,
                                                      dst)
             }
-            (CandidateApproval { candidate_id, client_auth, sections }, Section(_), Section(_)) => {
-                self.handle_candidate_approval(candidate_id, client_auth, sections, outbox)
+            (CandidateApproval { candidate_id, client_auth, .. }, Section(_), Section(_)) => {
+                self.handle_candidate_approval(candidate_id, client_auth, outbox)
```

Comment:
I guess we could just use a hash of the sections.  As well as cutting down the message size, it would maybe be more obvious that we're not actually doing anything with the sections on receipt of this message.

Not a big deal though; this isn't a common message compared to many others.